rust/rsc/src/rsc/main.rs

```diff
@@ -407,6 +417,8 @@ mod tests {
     async fn create_test_store(
         db: &DatabaseConnection,
     ) -> Result<Uuid, Box<dyn std::error::Error>> {
+        database::create_dbonly_blob_store(db).await?;
```

Comment:
Adding the dbonly store does muddy the terminology a bit, but there is still only one "active" store.

The active store is just the store that, by default, a blob is pushed to. The dbonly store is always enabled but not the default.

Happy to use a different term if you have any suggestions

Reply:
Do we have a test that this default dbonly blob store is working properly?

Regarding naming it seems like `active` store makes sense from the perspective of someone pushing blobs, but from someone pulling blobs they may get results from "inactive" stores. `default` also doesn't solve this confusion. It seems like "push" or "write" or some verb like that would need to be joined with the other adjective, so "default_push_store" or "active_write_store" or things like this but I'm not sure any of those combinations actually provide clarity, and may impair understanding.

Reply:
> Do we have a test that this default dbonly blob store is working properly?

No not yet, I'll get one added in a followup. Testing the blob uploads is a bit more tricky since I need to stream files but it should be done